src/client.tsx

```diff
@@ -11,6 +12,12 @@ const render = (Component: React.FC) => {
 const initializeApp = () => {
     const App = require('./components/App/App').App;
 
+    const { DISABLE_GA, GA_TRACKING_ID } = env;
+
+    if (!DISABLE_GA) {
```

Comment:
This doesn't seem to work as intended.  ENV values of "false" or `false` both don't seem to fire this.